src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -25,17 +25,404 @@ namespace XBU = XBUtilities;
 #include "core/common/system.h"
 #include "core/common/device.h"
 #include "core/common/error.h"
+#include "core/pcie/common/device_pcie.h"
```

Comment:
This does not look right at all.   If xbmgmt2 is pcie specific then it should not live under core/tools

Reply:
Yes. It's not needed. Thanks for pointing it out